lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -416,6 +432,13 @@ class TextEditingElement {
       _onChange(_lastEditingState);
     }
   }
+
+  void _maybeSendAction(html.KeyboardEvent event) {
+    if (event.keyCode == 13) {
```

Comment:
Suggestion: We can add a documentation here for the keyCode. Or we can have a 

const int _kReturnKeyCode = 32; 

on the top of the file.